easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb

```
@@ -0,0 +1,49 @@
+easyblock = 'SCons'
+
+name = 'Cantera'
+version = '2.6.0'
+
+homepage = 'https://github.com/Cantera/cantera'
+description = """Chemical kinetics, thermodynamics, and transport tool suite"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/Cantera/cantera/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7273622ea76a53373cee820f939613b15eea3dd83db6e1b127c5ed043f77dc5b']
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('Boost', '1.79.0'),
+    ('SUNDIALS', '6.3.0'),
+    ('yaml-cpp', '0.7.0'),
+    ('ruamel.yaml', '0.17.21'),
+]
+builddependencies = [
+    ('SCons', '4.4.0'),
+    ('Eigen', '3.4.0'),
+    ('fmt', '9.1.0'),
+    ('googletest', '1.11.0'),
+]
+
+local_common_opts = 'env_vars=all CC="$CC" CXX="$CXX" cc_flags="$CFLAGS" cxx_flags="$CXXFLAGS" '
+# local_common_opts += 'blas_lapack_libs=openblas blas_lapack_dir=$BLAS_LAPACK_LIB_DIR '
+# local_common_opts += 'sundials_include=$EBROOTSUNDIALS/include sundials_libdir=$EBROOTSUNDIALS/lib'
+# local_common_opts += 'blas_lapack_libs=openblas blas_lapack_dir=${EBROOTFLEXIBLAS}/lib'
```

Could you explain why these lines have been commented out? According to the docs, they still seem relevant. Would the following work?

```suggestion
local_common_opts += 'blas_lapack_libs=flexiblas blas_lapack_dir=$BLAS_LAPACK_LIB_DIR '
local_common_opts += 'sundials_include=$EBROOTSUNDIALS/include sundials_libdir=$EBROOTSUNDIALS/lib'
```

---

Hello @bedroge sorry for the delay, but the final user never give me any feedback. The Cantera was build but empty without any module. 
I commented out the lines because it was giving me _Sanity check failed: Check for banned/required shared libraries_
Using your tip, solved the Sanity check problem. Reading the documentation of Canetera, https://www.cantera.org/compiling/installation-reqs.html#ubuntu-debian also **numpy** was needed so I added SciPy-bundle